include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -515,6 +515,29 @@ def TTIR_ReshapeOp: TTIR_DPSOp<"reshape"> {
     let hasVerifier = 1;
 }
 
+def TTIR_SliceOp: TTIR_DPSOp<"slice"> {
+    let summary = "Slice op.";
+    let description = [{
+      Extract a portion of a tensor based on the specified start (`begins`), stop (`ends`), and step
+      indices for each dimension.
+    }];
+
+    let arguments = (ins AnyRankedTensor:$input,
+                         AnyRankedTensor:$output,
+                         I32ArrayAttr:$begins,
+                         I32ArrayAttr:$ends,
```

Comment:
Nit: rename to `start` and `end`. This way it's aligned with TTNN lib's param names.

Reply:
As you already mentioned, that naming was used in the older version of TTNN.